examples/java/src/test/java/org/apache/beam/examples/complete/TfIdfTest.java

```diff
@@ -21,7 +21,6 @@
 //   name: TfIdfTest
 //   description: Unit-test for the TfIdf example.
 //   multifile: false
-//   pipeline_options:
```

Comment:
How is it related to default example?

Reply:
@daria-malkova per our discussion we decided to remove empty optional fields from tags in this PR.

Reply:
ok